web/app/assets/locales/locale-en.json

```diff
@@ -34,7 +34,8 @@
     "help": "Help",
     "locked_tip": "<br/>Click to login by unlocking your wallet.<br/><br/>Closing the browser window will also log you out.",
     "unlocked_tip": "<br/>Click to log out by locking your wallet.<br/><br/>Closing the browser window will also log you out.",
-    "account_notify": "The active account is now %(account)s"
+    "account_notify": "The active account is now %(account)s",
+    "account_value": "ACCOUNT VALUE"
```

Comment:
Please change ACCOUNT VALUE to normal lowercase "Account value"

Reply:
OK , updated